### ISSUE TITLE
Ｕｓｅｆｕl Blogs and Tutorials

### ISSUE THREAD

**Issue by yushenxiang** [yushenxiang/MachineLearningTips#30]
https://www.pyimagesearch.com/static/cv_dl_resource_guide.pdf
优秀博客和代码参考

**Comment by yushenxiang** [comment]
https://www.pyimagesearch.com/2018/05/07/multi-label-classification-with-keras/
keras多标签分类
https://www.pyimagesearch.com/2018/06/04/keras-multiple-outputs-and-multiple-losses/
keras多输出分类

**Comment by yushenxiang** [comment]
https://mp.weixin.qq.com/s?__biz=MzA3MzI4MjgzMw==&mid=2650747013&idx=3&sn=db30f433e3241cfdf696dc2466c2f772&chksm=871af4fbb06d7ded75430b0793573e8016b61efa012de5e1638e5b2345760d796c1a1b996996&mpshare=1&scene=1&srcid=0901bPaNbb2jLUXB8e8BCygy&pass_ticket=LURovF1%2BHYra3BgSQkWBWMG8NtNb8RMuGS81cjs3n%2FAgL1T4WD8qEayLpKjGsagx#rd
ｋｅｒａｓ多输出分类

**Comment by yushenxiang** [comment]
https://www.pyimagesearch.com/2018/12/31/keras-conv2d-and-convolutional-layers/
In the first part of this tutorial, we are going to discuss the parameters to the Keras Conv2D class.

From there we are going to utilize the Conv2D class to implement a simple Convolutional Neural Network.
